[2025-06-05T16:06:40.540+0000] {taskinstance.py:1946} INFO - Dependencies not met for <TaskInstance: dag_with_postgres_operator_v01.insert_into_table manual__2025-06-05T16:04:11.870740+00:00 [skipped]>, dependency 'Task Instance State' FAILED: Task is in the 'skipped' state.
[2025-06-05T16:06:40.551+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
[2025-06-05T16:07:32.754+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.insert_into_table manual__2025-06-05T16:04:11.870740+00:00 [queued]>
[2025-06-05T16:07:32.773+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.insert_into_table manual__2025-06-05T16:04:11.870740+00:00 [queued]>
[2025-06-05T16:07:32.774+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2025-06-05T16:07:32.791+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2025-06-05 16:04:11.870740+00:00
[2025-06-05T16:07:32.797+0000] {standard_task_runner.py:60} INFO - Started process 54341 to run task
[2025-06-05T16:07:32.802+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v01', 'insert_into_table', 'manual__2025-06-05T16:04:11.870740+00:00', '--job-id', '101', '--raw', '--subdir', 'DAGS_FOLDER/postgresql_dag.py', '--cfg-path', '/tmp/tmpobdlqmnt']
[2025-06-05T16:07:32.804+0000] {standard_task_runner.py:88} INFO - Job 101: Subtask insert_into_table
[2025-06-05T16:07:32.869+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator_v01.insert_into_table manual__2025-06-05T16:04:11.870740+00:00 [running]> on host 14aa2e8b0b7d
[2025-06-05T16:07:33.033+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v01' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2025-06-05T16:04:11.870740+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-05T16:04:11.870740+00:00'
[2025-06-05T16:07:33.035+0000] {sql.py:276} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2025-06-05', 'dag_with_postgres_operator_v01')
        
[2025-06-05T16:07:33.049+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-05T16:07:33.065+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-05T16:07:33.099+0000] {sql.py:450} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2025-06-05', 'dag_with_postgres_operator_v01')
        , parameters: None
[2025-06-05T16:07:33.106+0000] {sql.py:459} INFO - Rows affected: 1
[2025-06-05T16:07:33.123+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v01, task_id=insert_into_table, execution_date=20250605T160411, start_date=20250605T160732, end_date=20250605T160733
[2025-06-05T16:07:33.179+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-05T16:07:33.210+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
